dv/uvm/core_ibex/yaml/rtl_simulation.yaml

```diff
@@ -18,7 +18,7 @@
 # As a result, passing -fno-extended-identifiers tells G++ to pretend that
 # everything is ASCII, preventing strange compilation errors.
 - tool: vcs
-  env_var: IBEX_ROOT
+  env_var: IBEX_ROOT, EXTRA_COSIM_CFLAGS
```

Comment:
Do we also need to tweak some documentation in this commit to tell (some) people to pass this flag?

Reply:
We should document it but as it's only really relevant to CentOS 7 and there's other things that need documenting for that also I'd prefer to deal with that in a follow up PR (in particular so we can get the various fixes in this PR in).